TicketSystem/TicketSystem.Tests/UnitTests/ValidatorTests/ShortMessageValidatorTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FluentValidation.TestHelper;
+using TicketSystem.API.Validators;
+using TicketSystem.API.ViewModels.Messages;
+
+namespace TicketSystem.Tests.UnitTests.ValidatorTests;
+
+public class ShortMessageValidatorTest
+{
+    private readonly ShortMessageValidator _validator;
+
+    public ShortMessageValidatorTest()
+    {
+        _validator = new ShortMessageValidator();
+    }
+
+    [Theory]
+    [InlineData("H", null, 1)]
+    [InlineData("H", 1, 1)]
+    [InlineData("H", int.MaxValue, int.MaxValue)]
+    [InlineData("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", int.MinValue, int.MaxValue)]
+    [InlineData("Hello", null, 123456789)]
+    public async Task ShortMessageValidator_CorrectParameters_Success(string text, int? ticketId, int userId)
+    {
+        // Arrange
+        var shortMessage = new ShortMessageViewModel
+        {
+            Text = text,
+            TicketId = ticketId,
+            UserId = userId
+        };
+
+        // Act
+        var result = await _validator.TestValidateAsync(shortMessage);
+
+        // Assert
+        Assert.True(result.IsValid);
+    }
```

Comment:
Maybe better to use Shoudly or Fluent Assertion.

Reply:
![image](https://user-images.githubusercontent.com/92907361/220551120-9218011b-18f7-4663-b7a1-50b9da593dee.png)
You mean using something like this?

Or use a more detailed field check
result.ShouldHaveValidationErrorFor(x => x.Text);

Reply:
Something like:

result.IsValid.ShouldBeTrue();